axiom-eth/src/rlp/tests.rs

```diff
@@ -394,6 +439,46 @@ mod rlp {
         }
     }
 
+    #[test] // fail
```

Comment:
tests should always "pass"
if you want to test that the MockProver is failing, you can do `MockProver::run(..).verify().is_err()` to get a boolean for whether it didn't pass.